### PR TITLE
🚀 added release drafter

### PR DESCRIPTION
**Pull request by lalmei**

### CODE REVIEW

**Comment by cswarth** [comment]
Can you include an example of the release notes this produces here?
Are the release notes create manually or is it part of a CI step?

**Comment by lalmei** [comment]
As mention in DM  `.github/release-drafter.yml` is the drafter template, the great thing is that is adds to a draft that can be modified by hand before the release. Once the release is published the pypi is pushed

**Comment by lalmei** [comment]
the example is here btw https://github.com/release-drafter/release-drafter#example

File: .github/workflows/push_release.yml [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+      run: |
+        python -m pip install --upgrade pip
+        pip install setuptools wheel twine
+    - name: Build and publish
+      env:
+        TWINE_USERNAME: ${{ secrets.PYPI_USERNAME }}
+        TWINE_PASSWORD: ${{ secrets.PYPI_PASSWORD }}
+      run: |
+        python setup.py sdist bdist_wheel
+        twine upload dist/*
```
Comment (andyndang) [inline .github/workflows/push_release.yml]:
```suggestion
        twine upload dist/*

```

File: .github/workflows/release-drafter.yml [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+  update_release_draft:
+    runs-on: ubuntu-latest
+    steps:
+      # Drafts your next Release notes as Pull Requests are merged into "mainline"
+      - uses: release-drafter/release-drafter@v5
+        with:
+          # (Optional) specify config name to use, relative to .github/. Default: release-drafter.yml
+          # config-name: my-config.yml
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```
Comment (andyndang) [inline .github/workflows/release-drafter.yml]:
```suggestion
          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }} 

```

File: .github/release-drafter.yml [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+  - title: " 🚨 Testing"
+    labels: 
+      - "testing"
+      - "test"
+
+exclude-labels:
+  - "changelog: skip"
+
+template: |
+
```
Comment (andyndang) [inline .github/release-drafter.yml:31]: intentional blank line?

**Review by andyndang — CHANGES_REQUESTED** [review]
LGTM with minor changes

**Review by cswarth — APPROVED** [review]
LGTM

File: .github/release-drafter.yml (hunk 3 above)
Comment (lalmei) [reply .github/release-drafter.yml:31]: it was in the template I found, i haven't gone down to  the source code
```suggestion
```

**Review by andyndang — APPROVED** [review]
LGTM

**Pull request merged**